rebar.lock

```diff
@@ -1,4 +1,4 @@
-{"1.2.0",
```

Comment:
So you used an older version of rebar here?

Reply:
I have the latest release `3.13.2`. I think @tsloughter last ran it with an rc version of `3.14.0`.

https://github.com/erlang/rebar3/releases

Reply:
If you didn't do any dep upgrades/changes can you reset this file to the original.